trigger-openqa_in_openqa

```diff
@@ -68,6 +68,15 @@ create_devel_openqa_snapshot() {
     elif [[ -e job_post_skip_submission ]]; then
         rm job_post_skip_submission
     fi
+    echo "Checking if all relevant packages are build and published"
+    for package in $auto_submit_packages; do
+        local state
+        state=$($osc results -r openSUSE_Tumbleweed -a x86_64 --no-multibuild "$src_project" "$package")
+        if grep -q -v 'succeeded\|disabled' <<< "$state"; then
+            echo -e "Package $package is not yet ready for release\n$state" | tee job_post_skip_submission
+            return
+        fi
+    done
```

Comment:
This code needs to run *after* the snapshot creation and on the snapshot. Otherwise it still isn't guaranteed that packages are built and published when the snapshot is created. Of course this means the code needs to delete the problematic snapshot in the error case.

Reply:
Yes, that's exactly the problem I described in the linked progress ticket. This was intended as a quick hack without changing much. I plan to address this issue in a subsequent work.

Reply:
As I though about it a little, I still think I really want to make that check *before* telling osc to make the copy as I want to know the state of the package I am about to copy over (remember, we are not doing any rebuild in the target project). I reordered the code a little to minimize the window between checking the state and calling `osc release` which should help. As we have only one source package for openQA (and one for os-autoinst) it should be just a quick look and copy over for each of them.